jobs/cloud_controller_ng/spec

```diff
@@ -103,6 +103,10 @@ provides:
   - cc.internal_service_hostname
   - cc.tls_port
   - cc.mutual_tls.ca_cert
+- name: cloud_controller_networking_info
```

Comment:
Can we tweak the name a bit to make it more clear that this is for container networking and not general "I want to talk with the Cloud Controller" networking -- I've seen this cause confusion in the past with some of our links and their consumers.

Maybe something like:
* `cloud_controller_container_networking_info`
* `cloud_controller_app_networking_info`
* `cloud_controller_runtime_networking_info`
* Something even better 🥇